components/cd-grid/README.md

```diff
@@ -24,7 +24,20 @@ This would allow a header and footer span the full width inside a grid layout.
 @TODO This might be dropped in favour of more robust grid system.
 
 ## Caveats
-The selectors must be added to the parent div
+The selectors must be added to the parent div.
+
+### Using with Drupal Views Grid format
+The Views UI allows configuration for the number of columns, and alignment
+(horizontal or vertical). The `cd-grid` component is used **only when the
+Horizontal alignment is selected**. The checkbox for **Automatic width
+should be unchecked** and **maximum number of columns is 4**. See the template
```

Comment:
Just wondering if we should not add the `cd-grid` only when those 2 conditions are met (in addition to the `horizontal` alignement).